fbw-a380x/src/systems/instruments/src/MFD/pages/SURV/MfdSurvControls.tsx

```diff
@@ -250,21 +253,23 @@ export class MfdSurvControls extends DisplayComponent<MfdSurvControlsProps> {
               <div class="mfd-surv-controls-tcas-left">
                 <RadioButtonGroup
                   values={['TA/RA', 'TA ONLY', 'STBY']}
+                  onModified={(val) =>
+                    this.props.bus.getPublisher<MfdSurvEvents>().pub('tcas_alert_level', 2 - val, true)
+                  }
                   selectedIndex={this.tcasTaraSelectedIndex}
```

Comment:
It would be a lot safer to subscribe to `this.tcasTaraSelectedIndex`, in case it is also mutated by some other means. This has a high chance of being a bug in future.